lstmcpipe/stages/mc_dl1_to_dl2.py

```diff
@@ -228,7 +228,7 @@ def dl1_to_dl2(
 
         # sbatch --parsable --dependency=afterok:{wait_ids_proton_and_gammas} --wrap="{cmd}"
         batch_cmd = (
-            f"sbatch --parsable -p short --dependency=afterok:{wait_jobs} -J {job_name[particle]}"
+            f"sbatch --parsable -p short --mem 64G --dependency=afterok:{wait_jobs} -J {job_name[particle]}"
```

Comment:
Yeah, probably.
I ran into issues actually, but that seems to be because I used the lstchain default config from their repo.
That one has no event filters so the files and models get very big.